.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,45 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Awaiting resposnse issues
+        uses: actions/stale@v5
+        with:
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          stale-issue-label: "stale"
+          only-issue-labels: 
+           - stat:awaiting response
+          stale-issue-message: > 
+            This issue is stale because it has been open for 30 days with no activity.
+            It will be closed if no further activity occurs. Thank you.
+          close-issue-message: >
+            This issue was closed because it has been inactive for 14 days since being marked as stale.
+            Please reopen if you'd like to work on this further.
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          stale-pr-message: "This PR is stale because it has been open for 30 days with no activity."
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Contribution issues
+        uses: actions/stale@v5
+        with:
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
```

Comment:
This would be half a year until marking as stale (effectively ~2 releases) and a full year before closing. This is for issues we know are good to fix but cannot spend cycles in fixing so we ask community to work on.

Assumption is that in a year TF moves that the issue is no longer needed.